packages/spotlib/spotlib.4.0.2/opam

```diff
@@ -0,0 +1,26 @@
+opam-version: "1.2"
+version: "4.0.2"
+authors: "Jun Furuse"
+maintainer: "jun.furuse@gmail.com"
+homepage: "https://bitbucket.org/camlspotter/spotlib"
+bug-reports: "https://bitbucket.org/camlspotter/spotlib/issues?status=new&status=open"
+dev-repo: "hg://https://bitbucket.org/camlspotter/spotlib"
+build: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-build" ]
+]
+install: [
+  [ "ocaml" "setup.ml" "-install" ]
+]
+remove: [
+  [ "ocaml" "setup.ml" "-configure" "--prefix" prefix ]
+  [ "ocaml" "setup.ml" "-uninstall" ]
+]
+depends: [
+  "ocamlfind" { build }
+  "omake" { build & < "0.10" }
+  "ppx_test" { >= "1.5.0" }
+]
+available: [
+  ocaml-version >= "4.06.0"
```

Comment:
The opam in the source repository seem to allow ```>= 4.04.0``` and at least ```4.05.0``` seem to compile fine.
Is there a special thing that makes it unavailable under ```4.04``` and ```4.05``` ?

Reply:
@camlspotter ?